MyHeartCounts/Supporting Files/Info.plist

```diff
@@ -4,6 +4,63 @@
 <dict>
 	<key>CFBundleAllowMixedLocalizations</key>
 	<false/>
+	<key>NSSensorKitUsageDescription</key>
+	<string>My Heart Counts uses SensorKit to better TODO TODO TODO TODO</string>
+	<key>NSSensorKitPrivacyPolicyURL</key>
+	<string>https://myheartcounts.su.domains/privacy-policy/</string>
+	<key>NSSensorKitUsageDetail</key>
+	<dict>
+		<key>SRSensorUsageWristDetection</key>
+		<dict>
+			<key>Description</key>
+			<string>My Heart Counts uses On-Wrist Data to TODO TODO TODO</string>
```

Comment:
Probably needs to be updated once we know what we aim to collect. We an also transform this into an separate issue for now and do it later, similar to a few other things.